Makefile

```diff
@@ -35,7 +35,13 @@ $(STATIC_LIB): $(OBJECTS)
 %.o: %.cpp
 	$(CXX) -c -o $@ $< $(CXXFLAGS) -MMD
 
+install:
+	mkdir -p $(TARGET_DIR_BIN)
```

Comment:
Where is TARGET_DIR_BIN defined? This does not use Unix-like conventions. The Makefile isn't really supposed to be a robust build system for SPIRV-Cross. Is it possible to use CMake instead which should already install stuff?

Also, headers and stuff are not installed either, so the static lib is not particularly useful here.